paths/graphql/outline.md

```diff
@@ -0,0 +1,24 @@
+---
+
+---
+
+Top-level numbers individual pages?
+
+1. Introduction
+  - Motivation, present the problem that needs solving
```

Comment:
I know we briefly discussed this during the Hack House @hectorsector but if you can provide a very brief idea of what the student will accomplish by finishing the class, I should be able to create a baseline for the copy that the course will use.